tooling/sync/Makefile

```diff
@@ -19,132 +19,92 @@ else
 	DATA_PATH = ~/.local/share
 endif
 
-help: ## Display help for the makefile.
-	@grep -E '^[a-zA-Z0-9_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+# Set checkpoint sync url based on network
+NETWORK ?= mainnet# default network if not set
```

Comment:
IMO this should probably be `sepolia`. It's the closest to mainnet (in the sense of being pre-merge) without risking money.